src/framework/parsers/glb-parser.js

```diff
@@ -1112,17 +1112,22 @@ const extensionIridescence = (data, material, textures) => {
 const createMaterial = (gltfMaterial, textures) => {
     const material = new StandardMaterial();
 
+    if (gltfMaterial.hasOwnProperty('name')) {
+        material.name = gltfMaterial.name;
+    }
+
     // glTF doesn't define how to occlude specular
     material.occludeSpecular = SPECOCC_AO;
 
     material.diffuseVertexColor = true;
-
     material.specularTint = true;
     material.specularVertexColor = true;
 
-    if (gltfMaterial.hasOwnProperty('name')) {
-        material.name = gltfMaterial.name;
-    }
+    // Set glTF spec defaults
+    material.specular.set(1, 1, 1);
+    material.gloss = 1;
+    material.glossInvert = true;
```

Comment:
I'm not sure about the gloss being 1 here, I think it should be 0.
The [spec](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-material-pbrmetallicroughness): 
```
roughnessFactor | The factor for the roughness of the material. | default: 1
```

And so I think gloss needs to be zero, which makes the roughness 1 when glossInvert is enabled.
Before, in StandardMaterial, it would use this default value (close to 0)
```
    _defineFloat('gloss', 0.25);
```

Reply:
Agreed. Done.